api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -117,14 +117,29 @@
                                      baseTableName="patient_program_attribute" baseColumnNames="voided_by"
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
+	<changeSet id="20180501-TRUNK-5338" author="Kelechi+iheanyi">
+		<preConditions onFail="MARK_RAN">
+			<tableExists tableName="conditions"/>
+		</preConditions>
+		<comment>
+			Modify conditions table to match condition object in the core platform if the conditions table already exists in the database
+		</comment>
+		<renameColumn columnDataType="int" newColumnName="previous_version" oldColumnName="previous_condition_id" tableName="conditions"/>
+		<renameColumn columnDataType="varchar(255)" newColumnName="clinical_status" oldColumnName="status" tableName="conditions"/>
+		<renameColumn columnDataType="int" newColumnName="condition_coded" oldColumnName="concept_id" tableName="conditions"/>
+
+		<addColumn tableName="conditions">
```

Comment:
This should also be a separate changeset

Reply:
Okay. Noted